react-app/src/App.js

```diff
@@ -44,6 +49,9 @@ function App() {
           <Route path="/friends/:id">
             <FriendPage />
           </Route>
+
```

Comment:
please avoid leaving unnecessary line breaks - in the real world, you would be pushing your code via PRs to the "production-ready" codebase in dev/main - it's good practice to have a consistent coding style